rpcd/etc/openstack_deploy/env.d/nova.yml

```diff
@@ -0,0 +1,113 @@
+---
```

Comment:
I don't believe the patterns will work the same way outside of the `--limit` parameter or the `hosts` key in a playbook. I think, but don't know, that being in one of those two places gets special parsing before anything else is run.

My original suggestion for the pattern targeting was based on a wrong assumption, so clearly it won't work. Extending the when conditions of the ensure_local_tasks.yml file seems to be fraught with edge cases that will be a nightmare to maintain.

So using patterns doesn't actually simplify this as much as I thought/hoped it would. In that case, I think the environment file is probably more straightforward and less work, though as Jesse mentioned it could probably be pared down to target only the keys that changed, rather than the whole file.